etc/fetchers/fetch-wbw.ts

```diff
@@ -0,0 +1,42 @@
+import { toSecond } from "../../lib/string-utils";
+
+import { fetchDocs } from "./fetch-docs";
+import { fetchFaqSheets } from "./fetch-faq-sheets";
+import { fetchSheets } from "./fetch-sheets";
+
+import chalk from "chalk";
+import ora from "ora";
+
+(function fetchWbw() {
+  const start = process.hrtime();
+  const spinner = ora(`${chalk.yellowBright("Fetching all data...")}`).start();
+
+  fetchFaqSheets()
+    .then(() => {
+      const end = `${toSecond(process.hrtime(start))} seconds`;
+      spinner.succeed(`Fetching FAQ done in ${chalk.greenBright(end)}`);
+      spinner.start(`${chalk.yellowBright("Fetching next data...")}`);
+    })
+    .catch((err) => {
+      console.error(err);
```

Comment:
Can we add `chalk.red(err)` here?

Reply:
Solved by fee6f4f698fb9a9153742613218f0c18285869f8